commit c7751a36482daa5979c5a480eb4918f9dc902d47
Author: Ruben Maher <ruben@maher.fyi>
Date:   Fri Mar 29 14:20:00 2019 +0900

    Revert "Disable multicolor fonts on OS X since they are not supported on free systems"
    
    This reverts commit 9344612d3cd164317170b6189ec43175757e4231.

diff --git a/src/macfont.m b/src/macfont.m
index 59627823fa..d8e86201f5 100644
--- a/src/macfont.m
+++ b/src/macfont.m
@@ -2344,9 +2344,9 @@ So we use CTFontDescriptorCreateMatchingFontDescriptor (no
                   != (spacing >= FONT_SPACING_MONO)))
             continue;
 
-          /* Don't use a color bitmap font until it is supported on
-	     free platforms.  */
-          if (sym_traits & kCTFontTraitColorGlyphs)
+          /* Don't use a color bitmap font unless its family is
+             explicitly specified.  */
+          if ((sym_traits & kCTFontTraitColorGlyphs) && NILP (family))
             continue;
 
           if (j > 0
